### PR TITLE
Relation form update

### PR DESCRIPTION
**Pull request by anaion**
PR for #25975.

Please review and merge.

### CODE REVIEW

File: hstool/models.py [hunk 1]
```diff
@@ -109,15 +109,6 @@ class DriverOfChange(GenericElement):
...
 
-    def __unicode__(self):
-        return '{0}: {1} - {2} - {3} - {4}'.format(
-            self.name,
-            dict(DOC_TYPE_CHOICES).get(self.type, ''),
-            dict(DOC_TREND_TYPE_CHOICES).get(self.trend_type, ''),
-            dict(DOC_STEEP_CHOICES).get(self.steep_category, ''),
-            dict(DOC_TIME_HORIZON_CHOICES).get(self.time_horizon, ''),
-        )
-
```
Comment (alexef) [inline hstool/models.py]: Why remove?

File: hstool/tests/test_relations_views.py [hunk 2]
```diff
@@ -31,9 +31,9 @@ def test_default_fields_required(self):
     def test_successfully_added(self):
         form = self.resp.forms[0]
         form['source'].select(text=self.driver.name)
-        form['destination'].select(text=self.indicator.name)
         form['relationship_type'].select(text='Neutral relationship')
         form['description'] = 'description'
+        form['indicator'] = '2'
```
Comment (alexef) [inline hstool/tests/test_relations_views.py]: where is '2' coming from?

File: hstool/models.py (hunk 1 above)
Comment (anaion) [inline hstool/models.py]: b/c of the listing in the assessment view. Before this change all sources and destinations were Generic, so they were using the GenericElement unicode method.

File: hstool/tests/test_relations_views.py (hunk 2 above)
Comment (anaion) [inline hstool/tests/test_relations_views.py]: That's the id generated for the select in the IndicatorFactory.

**Comment by alexef** [comment]
👍

**Pull request merged**